packages/account/src/Sections/Security/ConnectedApps/__tests__/connected-apps.spec.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { render, screen, waitFor, waitForElementToBeRemoved, fireEvent } from '@testing-library/react';
+import ConnectedApps from '../connected-apps';
+import { act } from '@testing-library/react';
```

Comment:
imports from `@testing-library/react` can be merged in a single import statement.